dataduct/steps/scripts/create_load_redshift_runner.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+"""
+Replacement for the load step to use the redshift COPY activity instead
```

Comment:
Didn't we decide that all strings will be on the same line?
Either way there is a inconsistency with `dataduct/config/credentials.py`.